.github/workflows/docs.yml

```diff
@@ -111,15 +111,15 @@ jobs:
       - uses: actions/setup-python@v3
         with:
           python-version: '3.11'
+      - name: Setup Rust toolchain
+        uses: ./.github/actions/setup
```

Comment:
Is this necessary?

Reply:
Yes. This setup will add cargo related configs which speed up python binding build from 6m to 4m.